lib/locale/de.yml

```diff
@@ -1,113 +1,115 @@
 de:
   tds_status_codes:
+    suggestion: 'Vorschlag'
+    fraud_notice: 'Transaktionen mit diesem Code können als betrügerisch angesehen werden.'
     fraudulent_codes:
-      '09': true
-      '10': true
-      '11': true
+      '09': true wahr
+      '10': true wahr
+      '11': true wahr
```

Comment:
It must be a misunderstanding. I guess the translator thought that this should be translated too.

As far as I'm concerned the library should still work despite this mistake, because we check for truthiness (and a `String` is always truthy).

Nevertheless, I think this change should be reverted.